django/cantusdb_project/main_app/forms.py

```diff
@@ -515,6 +509,7 @@ class Meta:
         ]
         widgets = {
             "title": TextInputWidget(),
+            "rism_siglum": autocomplete.ModelSelect2(url="rismsiglum-autocomplete"),
```

Comment:
from the `SourceCreateForm`, you removed
```python
rism_siglum = forms.ModelChoiceField(
        queryset=RismSiglum.objects.all().order_by("name"), required=False
    )
```
Do you need to do the same below, within `SourceEditForm`?

Reply:
Yes, done! Thanks for finding that.